components/Common/AvatarGroup/index.module.css

```diff
@@ -0,0 +1,11 @@
+.avatarGroup {
+  @apply flex items-center overflow-x-auto;
+
+  .avatarRoot {
+    @apply -ml-2 first:ml-0;
+  }
+
+  .avatar {
+    @apply flex h-8 w-8 items-center justify-center rounded-full border-2 border-white bg-neutral-100 text-sm  dark:border-neutral-950 dark:bg-neutral-900;
```

Comment:
Please, re-read the collaborator guide. Especifically the Styling a Component part.

Reply:
"We require that you define one Tailwind Token per line, just as shown on the example above, since this improves readability" I missed that line really sorry. Can't the `npm run format` do it?

Reply:
Sadly nope. There's no eslint rule that can do that. We could create our own ESLint rule, but that might be overkill

Reply:
Ok thanks for clarification. I reread it twice so that I can see the changes. I think that we should notify changes to the collaborator guide on slack.